styles.css

```diff
@@ -1,32 +1,77 @@
 /* ****** GENERAL STYLES ******** */
+
 html {
-  background-color: #A9AAD2;
+  background-color: #EAEAF4;
```

Comment:
Not a 'NEED TO HAVE', but look into CSS variables for repeated values like colors, fonts, font sizes/weights. Makes it easier to change a color scheme without having to find each rule and manually changing, for example. Also nice for readability.